src/routes/asset.php

```diff
@@ -225,3 +225,30 @@
     'url' => 'asset/' . $args['id'],
   ], 200);
 });
+
+/*
+ * Delete asset from library
+ */
+$app->get('/asset/{id:[0-9]+}/delete', function ($request, $response, $args) {
```

Comment:
Ah, and, it should be `POST`, even if it doesn't require parameters -- mainly for security reasons though (as this would force CSRF check, and thus won't make people delete their assets by clicking random links by mistake).